setup.py

```diff
@@ -22,8 +32,8 @@
     name="nupic.audio",
     description=description,
     namespace_packages=["nupic"],
-    packages = find_packages(),
-    install_requires=requirements,
+    packages=find_packages(),
+    requires=findRequirements(),
```

Comment:
I replaced `install_requires` with `requires` because I could not figure out why it was throwing a `TypeError`. I'm not sure if this is the right way, but it is at least _closer_ to the right way?